machine/accton/accton_as7312_54x/busybox/conf/config

```diff
@@ -6,3 +6,7 @@ CONFIG_SYS_EEPROM_I2C_MEM_ADDR_BITS=8
 CONFIG_SYS_EEPROM_OFFSET=0
 CONFIG_SYS_EEPROM_SIZE=256
 CONFIG_SYS_EEPROM_MAX_SIZE=256
+CONFIG_MKSWAP=y
+CONFIG_FEATURE_MKSWAP_UUID=y
+CONFIG_SWAPONOFF=y
+CONFIG_FEATURE_SWAPON_PRI=y
```

Comment:
Is this something we should make available to every platform?

Seems a little strange to add it to just one platform.